### PR TITLE
Console - Clean angularjs the return

### PR DESCRIPTION
**Pull request by ThibaudAV**
## Issue

https://gravitee.atlassian.net/browse/APIM-3385

## Description

A small description of what you did in that PR.

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-pzrpiofivd.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/api/component/gio-connector-dialog/gio-connector-dialog.component.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @use 'sass:map';
-@use '@angular/material' as mat;
-@use '@gravitee/ui-particles-angular' as gio;
+@use '../../../../../node_modules/@angular/material/index' as mat;
```
Comment (loriepisicchio) [inline gravitee-apim-console-webui/src/management/api/component/gio-connector-dialog/gio-connector-dialog.component.scss]: Can we keep the previous syntax ?

File: gravitee-apim-console-webui/src/components/notification-settings/notification-settings-details/notification-settings-details.component.scss [hunk 2]
```diff
@@ -1,8 +1,8 @@
 @use 'sass:map';
 
-@use '@angular/material' as mat;
-@use '@gravitee/ui-particles-angular' as gio;
-@use '../../../../scss/gio-layout' as gio-layout;
+@use '../../../../node_modules/@angular/material/index' as mat;
+@use '../../../../node_modules/@gravitee/ui-particles-angular/index' as gio;
```
Comment (loriepisicchio) [inline gravitee-apim-console-webui/src/components/notification-settings/notification-settings-details/notification-settings-details.component.scss]: Keep previous import syntax

File: gravitee-apim-console-webui/src/components/notification-settings/notification-settings-list.component.scss [hunk 3]
```diff
@@ -1,8 +1,8 @@
 @use 'sass:map';
 
-@use '@angular/material' as mat;
-@use '@gravitee/ui-particles-angular' as gio;
-@use '../../../scss/gio-layout' as gio-layout;
+@use '../../../node_modules/@angular/material/index' as mat;
+@use '../../../node_modules/@gravitee/ui-particles-angular/index' as gio;
```
Comment (loriepisicchio) [inline gravitee-apim-console-webui/src/components/notification-settings/notification-settings-list.component.scss]: Keep previous syntax

File: gravitee-apim-console-webui/src/management/api/component/gio-connector-dialog/gio-connector-dialog.component.scss (hunk 1 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/api/component/gio-connector-dialog/gio-connector-dialog.component.scss]: aa intelij ...

**Review by jourdiw — APPROVED** [review]
LGTM

**Pull request merged**